Sources/MongoKitten/Database.swift

```diff
@@ -447,7 +448,7 @@ extension Database {
         // Check for success
         guard successDocument["ok"] as Int? == 1 else {
             logger.error("Authentication for MongoDB user \(details.username) with MongoCR failed against \(details.database) for the following reason")
-            logger.error(document)
+            logger.error((successDocument["errmsg"] as String?)!)
```

Comment:
I'd prefer this part to be reverted. LogKitten allows logging custom objects like BSON Documents which I personally find a *big* help.

The alternative is to log the document in debug and the message in error but even then we'll need the force unwrap gone.

`(successDocument["errmsg"] as String?)!` -> `successDocument["errmsg"] as String?) ?? ""`